common/datavalues/src/types/deserializations/mod.rs

```diff
@@ -36,6 +37,17 @@ pub trait TypeDeserializer: Send + Sync {
     fn de(&mut self, reader: &mut &[u8]) -> Result<()>;
     fn de_default(&mut self);
     fn de_batch(&mut self, reader: &[u8], step: usize, rows: usize) -> Result<()>;
+    fn de_batch_with_nullable(
```

Comment:
It's only for nullable, so it is not must to put this into trait.

Reply:
Yes, your idea is valuable.

Reply:
> It's only for nullable, so it is not must to put this into trait.
Yes, your idea is valuable.